make/examples/multiple-binaries/Makefile.test.log

```diff
@@ -1,4 +1,12 @@
 make build SOURCE_GIT_TAG=v42.43.44 SOURCE_GIT_COMMIT=aaa SOURCE_GIT_TREE_STATE=clean
+../../targets/openshift/operator/../yq.mk:9: warning: overriding recipe for target 'ensure-yq'
```

Comment:
This is not an expected output but I guess we haven't really transitively included a file with targets more then once yet. I hope this won't end with ifdefs, but it's the last resort.

Reply:
@tnozicka updated to yq.mk and yaml-patch.mk to avoid duplicate targets. Please let me know if this is what you had in mind.